### ISSUE TITLE
Capture One leftovers

### ISSUE THREAD

**Issue by DmitriyYukhanov** [Klocman/Bulk-Crap-Uninstaller#206]
Hey, recently I've deleted Capture One trial with BCU 4.5 version and here are the leftovers I found undetected after uninstallation:

Folders:
```
C:\ProgramData\Phase One (empty)
C:\Users\focus\AppData\Local\CaptureOne (has nested folders and files)
C:\Users\focus\AppData\Local\Phase_One (has nested folders and files)
```

Registry:

```
HKEY_LOCAL_MACHINE\SOFTWARE\Classes\.cocatalogdb
now has no values but had default string value "CaptureOneCatalog" before uninstallation

HKEY_LOCAL_MACHINE\SOFTWARE\Classes\.cocatalogtemplate
now has no values but had default string value "CaptureOneCatalogTemplate" before uninstallation

HKEY_LOCAL_MACHINE\SOFTWARE\Classes\.col
now has no values but had default string value "CaptureOneSession" before uninstallation

HKEY_LOCAL_MACHINE\SOFTWARE\Classes\.col45
now has no values but had default string value "CaptureOneSession" before uninstallation

HKEY_LOCAL_MACHINE\SOFTWARE\Classes\.col50
now has no values but had default string value "CaptureOneSession" before uninstallation

HKEY_LOCAL_MACHINE\SOFTWARE\Classes\.col60
now has no values but had default string value "CaptureOneSession" before uninstallation

HKEY_LOCAL_MACHINE\SOFTWARE\Classes\.comask
now has no values but had default string value "CaptureOneMask" before uninstallation

HKEY_LOCAL_MACHINE\SOFTWARE\Classes\.copreset
now has no values but had default string value "CaptureOnePreset" before uninstallation

HKEY_LOCAL_MACHINE\SOFTWARE\Classes\.cos
now has no values but had default string value "CaptureOneSettings" before uninstallation

HKEY_LOCAL_MACHINE\SOFTWARE\Classes\.cosessiondb
now has no values but had default string value "CaptureOneSession" before uninstallation

HKEY_LOCAL_MACHINE\SOFTWARE\Classes\.cosessiontemplate
now has no values but had default string value "CaptureOneSessionTemplate" before uninstallation

HKEY_LOCAL_MACHINE\SOFTWARE\Classes\.costyle
now has no values but had default string value "CaptureOneStyle" before uninstallation

HKEY_LOCAL_MACHINE\SOFTWARE\Classes\.costylepack
now has no values but had default string value "CaptureOneStylePackage" before uninstallation

HKEY_LOCAL_MACHINE\SOFTWARE\Classes\Applications\CaptureOne.exe 
had nested key with value with path to the capture one executable before uninstallation

HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\RADAR\HeapLeakDetection\DiagnosedApplications\CaptureOne.exe

HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Tracing\CaptureOne_RASAPI32

HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Tracing\CaptureOne_RASMANCS

HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows NT\CurrentVersion\Image File Execution Options\captureone.exe

HKEY_USERS\S-1-5-21-2081619299-1437623788-1181410189-1001\Software\Phase One
now empty but it had nested key 
HKEY_USERS\S-1-5-21-2081619299-1437623788-1181410189-1001\Software\Phase One\Capture One 
before uninstallation
```

**Comment by DmitriyYukhanov** [comment]
I believe leftovers scan may perform way better if it will make pre-process search before running an uninstallation to capture such cases.

**Comment by Klocman** [comment]
Added scanner for the `HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Tracing\` junk.

**Comment by Klocman** [comment]
Added scanner for the `HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\RADAR\HeapLeakDetection\DiagnosedApplications` junk.

What's left is the partially removed junk that didn't get picked up.

**Comment by Klocman** [comment]
Closing this issue since it's resolved except for one part that is handled in #162.
